hack/verify-all.sh

```diff
@@ -73,13 +73,6 @@ if [[ "${VERIFY_GOFMT:-true}" == "true" ]]; then
   cd "${REPO_PATH}"
```

Comment:
The GOLANGCI_LINT section below already runs gofmt, so the VERIFY_GOFMT section can be removed (along with the hack/verify-gofmt.sh script.

Reply:
The GOLANGCI_LINT section below already runs misspell, so the VERIFY_SPELLING section can be removed (along with the hack/verify-spelling.sh script.

Reply:
The GOLANGCI_LINT section below already runs whitespace, so the VERIFY_WHITESPACE section can be removed (along with the hack/verify-whitespace.sh script.